build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+git submodule update --init --recursive
+
+./bootstrap
+
+test -d build || mkdir build
+test -d build/debug || mkdir build/debug
+
+pushd build
+
+cmake -G Ninja -DCMAKE_BUILD_TYPE=Debug -B./Debug ..
+
+ninja -C Debug shards
```

Comment:
You can use:
```sh
cmake -Bbuild/Debug -G Ninja -DCMAKE_BUILD_TYPE=Debug
```
To not have to push the directory first (can just run from within the project folder)